Library/UIViewController-Preparation.swift

```diff
@@ -43,16 +44,21 @@ extension UIViewController {
     self.bindViewModel()
   }
 
+  internal func ksr_viewWillAppear(_ animated: Bool) {
+    self.ksr_viewWillAppear(animated)
+    self.bindStyles()
+  }
```

Comment:
oof, good catch. i could at the very least have this only bind once and then it should be a least a little more faithful to when it was only on `viewDidLoad`...